Source/FullScreenPokemon.ts

```diff
@@ -3358,6 +3369,21 @@ module FullScreenPokemon {
             }
         }
 
+        /**
+         * Activates a Detector to force the Player onto his/her bike and optionally to keep moving.
+         * 
+         * @param player   The Player.
+         * @param thing   A Detector triggered by player.
+         */
+        activateCyclingRoad(player: IPlayer, thing: ICyclingTriggerer): void {
```

Comment:
Forgot to nit this earlier - you're not activating CyclingRoad, you're activating a triggerer.